docs/making_queries.md

```diff
@@ -84,7 +84,7 @@ notes = await Note.objects.filter(Note.columns.id.in_([1, 2, 3])).all()
 Here `Note.columns` refers to the columns of the underlying SQLAlchemy table.
 
 !!! note
-    Note that `Note.columns` returns SQLAlchemy table columns, whereas `Note.fields` returns `orm` fields.
+Note that `Note.columns` returns SQLAlchemy table columns, whereas `Note.fields` returns `orm` fields.
```

Comment:
And based on [this](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) I think this isn't correct. What do you think?

Reply:
The same as above.